driver/web/docs/gen/gen_types.go

```diff
@@ -1,12 +1,12 @@
 // Package Def provides primitives to interact with the openapi HTTP API.
 //
-// Code generated by github.com/deepmap/oapi-codegen version v1.16.2 DO NOT EDIT.
```

Comment:
Please update the to latest tool version and then generate the struct. You use currently old version.